README.md

```diff
@@ -149,3 +151,33 @@ $ twine upload dist/*
 username: remme
 password: ********
 ```
+
+## Contributing
+
+Clone the project and install requirements:
+
+```bash
+$ git clone https://github.com/Remmeauth/remme-core-cli && cd remme-core-cli
+$ pip3 install -r requirements.txt
```

Comment:
Why not like this `pip3 install -r requirements.txt -r requirements-dev.txt -r requirements-tests.txt`?

Reply:
@Alladin9393, done.